README.md

```diff
@@ -79,51 +98,186 @@ Example:
 | updatedAt   | `Date`    | Last time the monitor was updated        |
 | CreatedAt   | `Date`    | When the monitor was updated             |
 
+</details>
+
+---
+
+<details>
+<summary><code>POST</code> <b>/api/v1/auth/register</b></summary>
+
+##### Method/Headers
+
+> | Method/Headers | Value            |
+> | -------------- | ---------------- |
+> | Method         | POST             |
+> | content-type   | application/json |
+
+##### Body
+
+> | Name      | Type     | Notes               |
+> | --------- | -------- | ------------------- |
+> | firstname | `string` |                     |
+> | lastname  | `string` |
+> | email     | `string` | Valid email address |
+> | password  | `string` | Min 8 chars         |
+
+##### Response Payload
+
+> | Type   | Notes      |
+> | ------ | ---------- |
+> | `User` | User model |
+
+##### Sample CURL request
+
+```
+curl --request POST \
+  --url http://localhost:5000/api/v1/auth/register \
+  --header 'Content-Type: application/json' \
+  --data '{
+	"firstname" : "User First Name",
+	"lastname": "User Last Name",
+	"email" : "user@gmail.com",
+	"password": "user_password"
+}'
+```
+
+##### Sample Response
+
+```json
+{
+  "success": true,
+  "msg": "User created}",
+  "data": {
+    "_id": "6645079aae0b439371913972",
+    "firstname": "User First Name",
+    "lastname": "User Last Name",
+    "email": "user@gmail.com",
+    "isActive": true,
+    "isVerified": false,
+    "updated_at": "2024-05-15T19:06:02.720Z",
+    "created_at": "2024-05-15T19:06:02.720Z",
+    "__v": 0
+  }
+}
+```
+
+</details>
+
 ---
 
-##### GET /api/v1/monitors
```

Comment:
Just a minor adjustment to the readme as the the path should be "/api/v1/auth" rather than "/api/v1" according to the code:
```javascript
app.use("/api/v1/auth", authRouter);
```

Reply:
Thanks for catching that, will update and commit

Reply:
On second glance, I checked out the readme and it appears to be correct :thinking: 

There's two routers right now, `auth` and `monitors`

```
app.use("/api/v1/auth", authRouter);
app.use("/api/v1/monitors", monitorRouter);
```

The auth routes should be

`/api/v1/auth`

and the monitor routes should be

`/api/v1/monitors/`


For example login is

`/api/v1/auth/login`

the endpoint for all monitors is

`/api/v1/monitors`

and the endpoint for a specific monitor is

`/api/v1/monitors/{monitorId}`

Can you double check and let me know?  Thanks!

Reply:
Yup that is correct!